shell/pages/c/_cluster/apps/charts/install.vue

```diff
@@ -837,15 +844,16 @@ export default {
 
       if (hasPermissionToSeeProvCluster) {
         const mgmCluster = this.$store.getters['currentCluster'];
+        const provClusterId = mgmCluster?.provClusterId;
         let provCluster;
 
         try {
-          provCluster = mgmCluster?.provClusterId ? await this.$store.dispatch('management/find', {
+          provCluster = provClusterId ? await this.$store.dispatch('management/find', {
```

Comment:
Not a huge blocker but I'm not a huge fan of this as a ternary. 

I think it'd be a lot cleaner if we used an if/else block to assign provCluster or if we initialized provCluster an empty object and then used an if block inside the try/catch to assign it the 'management/find' action.

I'd like it even better if we could skip the async/await and just fire off the `management/find` action from here and use `byId` getter in a computed property to return the appropriate value reactively when the store mutates upon completion of the API call.

Reply:
That second point is definitely the best, making async requests together up front would speed up load time.

For this PR though I don't think we should refactor too much. I've added this as a request in the issue to better handle missing info (as that will involve a fair bit of change) - https://github.com/rancher/dashboard/issues/9452#issuecomment-1662167610